tests/plugins/easyimage/uploadintegrations.js

```diff
@@ -1,7 +1,7 @@
 /* bender-tags: editor, clipboard, upload */
 /* bender-ckeditor-plugins: sourcearea, wysiwygarea, easyimage */
 /* bender-include: %BASE_PATH%/plugins/clipboard/_helpers/pasting.js, %BASE_PATH%/plugins/imagebase/features/_helpers/tools.js */
-/* global imageBaseFeaturesTools, pasteFiles, assertPasteEvent */
+/* global imageBaseFeaturesTools, assertPasteEvent */
```

Comment:
There are some problems with tests on IE8:
![image](https://user-images.githubusercontent.com/6498567/150792602-9677df64-6a9c-4050-bc27-cef7b1e1f589.png)

I think it is related to ignoring tests on unsupported environment:
```js
isSupportedEnvironment: function() {
			return !CKEDITOR.env.ie || CKEDITOR.env.version >= 11;
		},
```

Could you look at them?

Reply:
It seems that `bender.ignore()` does not stop executing the rest of `init()` function. I've replaced it with simple early return and added additional `assert.ignore()` inside `setUp()` function.